deploy/osd-machine-api/011-machine-api.srep-worker-healthcheck.MachineHealthCheck.yaml

```diff
@@ -21,3 +21,4 @@ spec:
     timeout: "480s"
     status: "Unknown"
   maxUnhealthy: 3
+  nodeStartupTimeout: 25m
```

Comment:
What is the default node start up time?

Reply:
Never mind its 10 minutes. https://github.com/openshift/machine-api-operator/blob/128c5c90918c009172c6d24d5715888e0e1d59e4/pkg/controller/machinehealthcheck/machinehealthcheck_controller.go#L47